spec/system/placements/support/schools/placements/support_user_views_placements_spec.rb

```diff
@@ -86,18 +106,29 @@ def and_i_dont_see_placements_from_another_school
     expect(page).not_to have_content(placement3.mentors.map(&:full_name).to_sentence)
   end
 
+  def and_the_placement_has_terms(placement)
+    term = create(:placements_term, :autumn)
+    placement.terms << term
+  end
```

Comment:
Personal preference: (Feel free to ignore) I would declare the term outside of the method and pass in a `terms` argument. This is for re-usability later.